### DISCUSSION TITLE
Helping with broken link on EPOCH website

### DISCUSSION THREAD

**Discussion by cooluser26** [epochpic/epoch#633] [Q&A] [answered]
Hi I am interested in learning more about the delta-f method. Could the link "to the pdf" be fixed here https://epochpic.github.io/documentation/code_details/using_delta_f ? 

Thank you for your help!

**Comment by Status-Mirror** [comment, marked as answer]
Hey @wnguyen1312,

Looks like this was an error converting from our old PDF files to the new web design. I don't think Hugo supports attachment of PDFs, so I've simply uploaded the contents onto the bottom of [the page](https://epochpic.github.io/documentation/code_details/using_delta_f). The delta-f documentation we have comes from these slides by Ben McMillan.

Quick warning that my ability to provide further delta-f support may be limited, as I think some of the delta-f developers aren't around anymore, and it's not a functionality that I have used myself.

Cheers,
Stuart

**Reply by cooluser26** [reply]
Thank you Stuart!

**Reply by cooluser26** [reply]
Hi Stuart,

It wasn't super obvious to me but by testing various input files, I found that the delta-f method would only run if the temperature of a species is non-zero (even if delta-f isn't applied to that particular species). Maybe it would be nice to say this somewhere in the documentation.

Thanks for all the help,

William.

**Reply by Status-Mirror** [reply]
Hey William,

I actually think you know more about the delta-f method in EPOCH than I do now! If you have a working input deck, could you attach it here? I may be able to add some basic demos for this in the future.

Cheers,
Stuart

**Reply by cooluser26** [reply]
Yes, here is my working deck for the delta-f method, which I base mainly on the 1D two-stream delta-f example (I think essentially you only need to add 3 or 4 lines to each species that you want to use delta-f on).
[delta_f-2.txt](https://github.com/Warwick-Plasma/epoch/files/14351650/delta_f-2.txt)
